### ISSUE TITLE
ECMA standard

### ISSUE THREAD

**Issue by wmleler** [dart-lang/site-webdev#635]
I know this might sound silly, but we really should mention the fact that Dart is an ECMA standard on the Dart website. Of course, before we do this, we should probably update the standard, since the most recent version is from 2015.

**Comment by filiph** [comment]
Fair point

/cc @kasperl -- how's the ECMA standard process these days?

**Comment by chalin** [comment]
Good point, but I think that this should be open over https://github.com/dart-lang/site-www.

cc @kwalrath @Sfshaza

**Comment by kwalrath** [comment]
Thanks, @chalin.
